### ISSUE TITLE
VMAgent can't scrape application without header: Accept: text/plain

### ISSUE THREAD

**Issue by to4kin** [VictoriaMetrics/VictoriaMetrics#608]
**Describe the bug**
One of my springboot applications return metrics only if the correct header set:

curl -v localhost:9080/actuator/prometheus

```
* Connected to localhost (::1) port 9080 (#0)
> GET /actuator/prometheus HTTP/1.1
> Host: localhost:9080
> User-Agent: curl/7.64.0
> Accept: */*
>
< HTTP/1.1 406
< Content-Length: 0
< Date: Tue, 07 Jul 2020 15:55:26 GMT
<
* Connection #0 to host localhost left intact
```

curl -v -H "Accept: text/plain" localhost:9080/actuator/prometheus

```
* Connected to localhost (::1) port 9080 (#0)
> GET /actuator/prometheus HTTP/1.1
> Host: localhost:9080
> User-Agent: curl/7.64.0
> Accept: text/plain
>
< HTTP/1.1 200
< Content-Type: text/plain; version=0.0.4;charset=utf-8
< Content-Length: 17870
< Vary: Accept-Encoding
< Date: Tue, 07 Jul 2020 15:56:28 GMT
<
```

curl http://vmagent-host:8429/targets

```
state=down, endpoint=http://<IP:PORT>/actuator/prometheus, labels={application="NAME", instance="INSTANCE-NAME", job="JOB-NAME"}, last_scrape=24.894s ago, scrape_duration=0.005s, error="unexpected status code returned when scraping \"http://<IP:PORT>/actuator/prometheus\": 406; expecting 200; response body: \"\""
```

VMAgent cannot scrape these metrics when prometheus can

**To Reproduce**
Same issue in Prometheus: https://github.com/prometheus/client_java/pull/267

**Expected behavior**
VMAgent should scrape this application too

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
$ ./victoria-metrics-prod --version
vmagent-20200702-100247-tags-v1.37.4-cluster-0-g0f754bea4
```

**Used command-line flags**
/vmagent-prod -remoteWrite.url=http://victoria-metrics-vminsert:8480/insert/0/prometheus/  -promscrape.consulSDCheckInterval=30s -promscrape.suppressScrapeErrors -promscrape.config=/vmagent-remotewrite-data/scrape.yml

**Comment by valyala** [comment]
@to4kin , the issue should be fixed in the commit 87f8c728bf9a8dae78d461808efdad27c2d45718 . Could you build `vmagent` from this commit according to [these docs](https://victoriametrics.github.io/vmagent.html#how-to-build-from-sources) and verify whether it now properly scrapes Spring Boot servers?

**Comment by valyala** [comment]
@to4kin , the bugfix has been included in [v1.38.0 release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed. Feel free re-opening if v1.38.0 or newer release of `vmagent` doesn't work as expected for your case.
